packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -137,6 +102,10 @@ class UpdatePackagesCommand extends FlutterCommand {
         defaultsTo: false,
         negatable: false,
       );
+    assert(
```

Comment:
I don't know that we ever run this command with asserts. Perhaps this should be an explicit integration test?

Reply:
Yeah, that makes sense. I did an assert because it's private but that's dumb, I should just make it public and test it.

Reply:
done